### DISCUSSION TITLE
希望增加 智谱清言 的翻译 API 自定义选项

### DISCUSSION THREAD

**Discussion by imspider** [immersive-translate/immersive-translate#1449] [Ideas]
### 新功能 | New Feature

1.希望增加 智谱清言 的翻译 API 自定义选项
2.
3.

### 为什么您想要添加这些功能 | Why you want to add these features

1.智谱清言 的翻译水平很高，而且速度很快
2.
3.

### 提议 | Proposal

桌面端的翻译软件 pot 就有智谱清言翻译选项，我使用过，感觉非常不错。

**Comment by keyboard3** [comment]
智谱的API，我们测试过，翻译一篇外网的文章，36%的内容提示内容敏感，不允许翻译...
